interpreter/cling/lib/MetaProcessor/MetaSema.cpp

```diff
@@ -341,9 +341,9 @@ namespace cling {
       "   " << metaString << "O <level>\t\t\t- Sets the optimization level (0-3)"
                              "\n\t\t\t\t  (not yet implemented)\n"
       "\n"
-      "   " << metaString << "class <name>\t\t- Prints out class <name> in a CINT-like style\n"
+      "   " << metaString << "class <name>\t\t- Prints out class <name> in a Cling-like style\n"
       "\n"
-      "   " << metaString << "files \t\t\t- Prints out some CINT-like file statistics\n"
+      "   " << metaString << "files \t\t\t- Prints out some Cling-like file statistics\n"
```

Comment:
Given the context, `CINT-like` look like legitimate uses of CINT (referring to some output that CINT generated).

Reply:
Reverted